_data/releases.tsv

```diff
@@ -45,3 +45,4 @@ Date	Data release	Code version	Sets	Wells	Experiments	Images	Planes	Size (TB)	Fi
 2020-10-06	prod89	0.8.9	14424	1286147		9903464	70616509	186.92806861126402	24.170604	433.145737887
 2020-11-19	prod90	0.9.0	15677	1746563		10646676	72846145	192.92555930017602	26.418725	472.513680031
 2020-12-10	prod91	0.9.1	15696	1753475		10708889	73157773	196.00307508509002	26.792054	476.26
+2021-01-26	prod92	0.9.2	17211	1753475	5001774830	76708943	283.4293715726564	28.162314	520.03
```

Comment:
`196 TB` -> `283 TB` does not feel completely right

Reply:
True :-) I think there was a column shift in the studies.tsv.... Fixing.